build/docker/zap-baseline.py

```diff
@@ -376,16 +391,16 @@ def main(argv):
 
       # print out the info rules
       for key, alert_list in sorted(alert_dict.iteritems()):
-        if config_dict.has_key(key) and config_dict[key] == 'INFO':
+        if (config_dict.has_key(key) and config_dict[key] == 'INFO')  or (not config_dict.has_key(key)) and info_unspecified:
           user_msg = ''
           if key in config_msg:
             user_msg = config_msg[key]
-          print_rule(config_dict[key], alert_list, detailed_output, user_msg)
+          print_rule('INFO', alert_list, detailed_output, user_msg)
           info_count += 1
 
       # print out the warning rules
       for key, alert_list in sorted(alert_dict.iteritems()):
-        if (not config_dict.has_key(key)) or (config_dict[key] == 'WARN'):
+        if (not config_dict.has_key(key) and not info_unspecified) or (config_dict.has_key(key) and config_dict[key] == 'WARN'):
```

Comment:
I dont like levels[3] - that means nothing to me when I read the code. I'm ok with either 'WARN' or LEVEL_WARN, but I dont really know whats considered best practice for python code :/